src/test/java/LoginTests.java

```diff
@@ -8,35 +9,37 @@
 public class LoginTests extends BaseTest {
 
     //    @Test(enabled = true, priority = 0, description = "LoginEmptyEmailPasswordTest")
-    @Test(dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
-    public static void loginEmptyEmailPasswordTest (String email, String password) {
-
-        login(email, password);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
+//    @Test(dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
+//    public static void loginEmptyEmailPasswordTest (String email, String password) {
+//        //login(email, password);
+//        LoginPage loginPage = new LoginPage(getDriver());
+//        loginPage.provideEmail("demo@class.com");
+//        loginPage.providePassword("");
+//        loginPage.clickSubmitBtn();
+//        //wait.
+//        Assert.assertEquals(driver.getCurrentUrl(), url);
+//    }
 
     //Page Object Model example
     @Test
     public void LoginValidEmailPasswordTest () {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public void loginEmptyEmailPasswordTest ()`